### DISCUSSION TITLE
[!] Redo NGC/IC, at least for these few DSO

### DISCUSSION THREAD

**Discussion by silas1037** [Stellarium/stellarium#2185] [General]
## 1. The few DSO ☆☆☆
I wrote an article about the most confusing NGC/IC object at the year before last year on [here](https://zhuanlan.zhihu.com/p/193128173), which already mentions few important objects with error positions.
```
NGC2185
NGC6664
NGC2252
NGC1513
NGC2304
NGC6222
NGC6383
```
The correction is very neccessary because some like Santa's Sleigh Cluster, NGC6664 are marked in error place, causing confusion to observers.
![image](https://user-images.githubusercontent.com/36977717/150096121-cb0081fa-c153-4076-be15-96341c5b3a2e.png)

You can find positions data for them in Dr. Wolfgang Steinicke's NI2021 here.
http://www.klima-luft.de/steinicke/index_e.htm

I had contact the author and get permission.
![image](https://user-images.githubusercontent.com/36977717/150100603-7a6527a2-e266-4e98-988b-b0a73d39ba25.png)

## 2.The further ☆

The NGC/IC catalogs are the most most important one for our amateur astronomer, so we should take care of them.
Dr. Wolfgang Steinicke release the second version, NI2021 at 2 February 2021. But how to use NI2021 into stellarium is still a problem, for so much data to solved again, such as incorrect cross-catalog. It may cost much time, but I would like to help for it.

refer:
#996
http://www.klima-luft.de/steinicke/ngcic/ngcic_e.htm
https://astronomy-mall.com/Adventures.In.Deep.Space/steve.ngc.htm
http://cseligman.com/text/atlas/ngc00.htm#ngcic

**Comment by silas1037** [comment]
@alex-w @gzotti

**Reply by gzotti** [reply]
@alex-w I have no objection against using Steinicke's version of NGC/IC. Of course, it must then be acknowledged in SUG H.5.

**Comment by alex-w** [comment]
I'll validate coordinates for these 7 NGCs

**Comment by silas1037** [comment]
1. It seems that the catalog.txt inside repo is not completely matching catalog.dat after I checked some objects? Such as some cross-catalog identifications are different. I would like to know whether that is a final txt of catalog.

2. I already checked all NGC/IC Objects whose priority <= 10 in [NI_rank](https://github.com/silas1037/DeepSkyCatalogs/releases/download/v1.0/NI_rank.xlsx) (a priority list of 13226 NGC/IC DSO compiled from many popular obseving list by me), and now the positions of them in stellarium's catalog are correct, just except what I mentioned above. Thus, it is not so important to correct the cross-id of other unknown NGC/IC DSO, also because `the term leaves open whether the object is non-existent or perhaps real at another place.` (*Observing and Cataloguing Nebulae and Star Clusters*). 

So we should leave these laborious work, anyone who wants to understand the meaning of these NGC/IC codes should go to their history. It's been a century since NGC/IC was released, and its fixing work is not quite done yet, that's too bad.

As for Steinicke's version of NGC/IC, I've also found bugs for it, so I don't have full confidence in his work yet.

![image](https://user-images.githubusercontent.com/36977717/154133034-18eff773-321c-411c-ad48-c768c2a1659c.png)
